libmscore/durationtype.cpp

```diff
@@ -414,7 +414,7 @@ Fraction TDuration::fraction() const
       }
 
 // Longest TDuration that fits into Fraction. Must fit exactly if truncate = false.
-TDuration::TDuration(const Fraction& l, bool truncate, int maxDots, DurationType maxType)
+TDuration::TDuration(const Fraction& l, bool /*truncate*/, int maxDots, DurationType maxType)
```

Comment:
Actually truncate is used in the QAssert, so a QUnused might be the better choice, maybe combined with NDEBUG